packages/kmod-5.15-nvidia/nvidia-fabricmanager.cfg

```diff
@@ -0,0 +1,34 @@
+# Modern, systemd-aware settings:
+# - Log to journal via stderr
+# - Keep running in the foreground
+LOG_LEVEL=4
+LOG_FILE_NAME=
+DAEMONIZE=0
+
+# Use Unix domain sockets instead of localhost ports.
+UNIX_SOCKET_PATH=/run/nvidia/fabricmanager.sock
+FM_CMD_UNIX_SOCKET_PATH=/run/nvidia/fabricmanager-cmd.sock
+
+# Start Fabric Manager in bare metal or full pass through virtualization mode.
+FABRIC_MODE=0
+FABRIC_MODE_RESTART=0
+
+# Terminate on NVSwitch and GPU config failure.
+FM_STAY_RESIDENT_ON_FAILURES=0
+
+# When there is a GPU to NVSwitch NVLink failure, remove the GPU with the failure
+# from NVLink P2P capability.
+ACCESS_LINK_FAILURE_MODE=0
+
+# When there is an NVSwitch to NVSwitch NVLink failure, exit Fabric Manager.
+TRUNK_LINK_FAILURE_MODE=0
+
+# When there is a NVSwitch failure or an NVSwitch is excluded, abort Fabric Manager.
+NVSWITCH_FAILURE_MODE=0
+
+# When Fabric Manager service is stopped or terminated, abort all running CUDA jobs.
+ABORT_CUDA_JOBS_ON_FM_EXIT=1
+
+# Path to topology and database files.
+TOPOLOGY_FILE_PATH=/usr/share/nvidia/nvswitch
+DATABASE_PATH=/usr/share/nvidia/nvswitch
```

Comment:
Need to fix the missing newlines at the end of all these files.

Reply:
changed as suggested.